content/en/docs/concepts/security/index.md

```diff
@@ -632,331 +628,260 @@ authorization result, `ALLOW` or `DENY`.
+configuration object. Like other Istio configuration objects, it is defined as Kubernetes `CustomResourceDefinition` [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+object.
 
-- **`ServiceRole`** defines a group of permissions to access services.
-- **`ServiceRoleBinding`** grants a `ServiceRole` to particular subjects, such
-  as a user, a group, or a service.
+An `AuthorizationPolicy` includes a `selector` and a list of `rules`, the `selector` specifies
+**where** to apply the policy and the `rules` specify **who** is allowed to do **what** under which **conditions**.
+Specifically:
 
-The combination of `ServiceRole` and `ServiceRoleBinding` specifies: **who** is
-allowed to do **what** under **which conditions**. Specifically:
+- **where** refers to the `selector` section in the `AuthorizationPolicy`.
+- **who** refers to the `from` section in the `rule` of the `AuthorizationPolicy`.
+- **what** refers to the `to` section in the `rule` of the `AuthorizationPolicy`.
+- **conditions** refers to the `when` section in the `rule` of the `AuthorizationPolicy`.
 
-- **who** refers to the `subjects` section in `ServiceRoleBinding`.
-- **what** refers to the `permissions` section in `ServiceRole`.
-- **which conditions** refers to the `conditions` section you can specify with
-  the [Istio attributes](/docs/reference/config/policy-and-telemetry/attribute-vocabulary/)
-  in either `ServiceRole` or `ServiceRoleBinding`.
+Workload selector decides where to apply the authorization policy. If not set,
+the authorization policy will be applied to all workloads in the same namespace
+as the authorization policy.
 
-#### `ServiceRole`
-
-A `ServiceRole` specification includes a list of `rules`, AKA permissions.
 Each rule has the following standard fields:
 
-- **`services`**: A list of service names. You can set the value to `*` to
-  include all services in the specified namespace.
+- **`from`**: A list of sources for specifying the identities.
 
-- **`methods`**: A list of HTTP methods. You can set the value to `*` to include all
-  HTTP methods. This field should not be set for TCP and gRPC services.
+- **`to`**: A list of operations for specifying the operations.
 
-- **`paths`**: HTTP paths or gRPC methods. The gRPC methods must be in the
-   form of `/packageName.serviceName/methodName` and are case sensitive.
+- **`when`**: A list of conditions for specifying extra conditions.
 
-A `ServiceRole` specification only applies to the namespace specified in the
-`metadata` section. A rule requires the `services` field and the other fields are optional.
-If you do not specify a field or if you set its value to `*`, Istio applies the field to all instances.
+An `AuthorizationPolicy` specification only applies to the namespace specified in the
+`metadata` section. All fields in the rule are optional. If you do not specify a field
+or if you set its value to `*`, Istio applies the field to all instances.
```

Comment:
There is a difference between leaving a field unset vs. set a field to `*`. 

* If you leave a field unset, there is no restriction on the field (can be empty). 
* If you set a field to "*", the field has to be non-empty. 

We need a section about exact match, prefix match, suffix match, presence match. See the [section](https://docs.google.com/document/d/1diwa9oYVwmLtarXb-kfWPq-Ul3O2Ic3b8Y21Z3R8DzQ/edit#bookmark=id.j08yqyvjd0nn) in design doc.

Reply:
Added a new section `Value matching` for this.